### ISSUE TITLE
Quantities notebook in quarantine

### ISSUE THREAD

**Issue by eblur** [astropy/astropy-tutorials#447]
As of PR #446, the quantities notebook appears out-of-date enough to [break Circle CI](https://circleci.com/gh/astropy/astropy-tutorials/724?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link).

We need a PR to bring it up to date or move it to the quarantine folder.

**Comment by kakirastern** [comment]
Yeah, let me take a look at it tomorrow

**Comment by kakirastern** [comment]
The `quantities` notebook is passing the CI checks now, but not the `vo` notebook: 
```
Configuration error:
There is a programmable error in your configuration file:

Traceback (most recent call last):
  File "/home/circleci/project/venv/lib/python3.7/site-packages/sphinx/config.py", line 319, in eval_config_file
    execfile_(filename, namespace)
  File "/home/circleci/project/venv/lib/python3.7/site-packages/sphinx/util/pycompat.py", line 81, in execfile_
    exec(code, _globals)
  File "/home/circleci/project/tutorials/conf.py", line 164, in <module>
    process_notebooks(nb_tutorials_path, **processkwargs)
  File "/home/circleci/project/scripts/convert.py", line 238, in process_notebooks
    nbc.execute()
  File "/home/circleci/project/scripts/convert.py", line 97, in execute
    executor.preprocess(nb, {'metadata': {'path': self.path_only}})
  File "/home/circleci/project/venv/lib/python3.7/site-packages/nbconvert/preprocessors/execute.py", line 405, in preprocess
    nb, resources = super(ExecutePreprocessor, self).preprocess(nb, resources)
  File "/home/circleci/project/venv/lib/python3.7/site-packages/nbconvert/preprocessors/base.py", line 69, in preprocess
    nb.cells[index], resources = self.preprocess_cell(cell, resources, index)
  File "/home/circleci/project/venv/lib/python3.7/site-packages/nbconvert/preprocessors/execute.py", line 448, in preprocess_cell
    raise CellExecutionError.from_cell_and_msg(cell, out)
nbconvert.preprocessors.execute.CellExecutionError: An error occurred while executing the following cell:
------------------
my_db = vos_catalog.get_remote_catalog_db(conf.conesearch_dbname)
my_cat = my_db.get_catalog_by_url(result.url + '&')
print(my_cat.dumps())
------------------


MissingCatalogTraceback (most recent call last)
<ipython-input-12-ada7f8616edf> in <module>
      1 my_db = vos_catalog.get_remote_catalog_db(conf.conesearch_dbname)
----> 2 my_cat = my_db.get_catalog_by_url(result.url + '&')
      3 print(my_cat.dumps())

~/project/venv/lib/python3.7/site-packages/astroquery/vo_conesearch/vos_catalog.py in get_catalog_by_url(self, url)
    258         if len(keys) < 1:
    259             raise MissingCatalog(
--> 260                 "No catalog with URL '{0}' found.".format(url))
    261 
    262         return VOSCatalog(self._catalogs[keys[0]])

MissingCatalog: No catalog with URL 'http://vizier.u-strasbg.fr/viz-bin/conesearch/I/220/out?&' found.
MissingCatalog: No catalog with URL 'http://vizier.u-strasbg.fr/viz-bin/conesearch/I/220/out?&' found.
```

Could be a server-side error.

**Comment by pllim** [comment]
Yeah... `http://gsss.stsci.edu/webservices/vo/ConeSearch.aspx?CAT=GSC23&` is down, so it grabbed `http://vizier.u-strasbg.fr/viz-bin/conesearch/I/220/out?` instead. If you remove the `+ '&'` then it would work but then if `gsss.stsci.edu` comes back online, then removing it would break. 😬

**Comment by pllim** [comment]
Oh wait... http://gsss.stsci.edu/webservices/vo/ConeSearch.aspx?CAT=GSC23& just came back online, it seems. 🤷

**Comment by pllim** [comment]
tl;dr -- If you rerun CI without any changes, it should pass now.

**Comment by kakirastern** [comment]
Thanks @pllim Let me rerun the tests now

**Comment by kakirastern** [comment]
Uh oh, now the `vo` notebook is causing another error thrown: 
```
INFO: Processing notebook conesearch.ipynb (in /home/circleci/project/tutorials/notebooks/vo) [convert]

Configuration error:
There is a programmable error in your configuration file:

Traceback (most recent call last):
  File "/home/circleci/project/venv/lib/python3.7/site-packages/sphinx/config.py", line 319, in eval_config_file
    execfile_(filename, namespace)
  File "/home/circleci/project/venv/lib/python3.7/site-packages/sphinx/util/pycompat.py", line 81, in execfile_
    exec(code, _globals)
  File "/home/circleci/project/tutorials/conf.py", line 164, in <module>
    process_notebooks(nb_tutorials_path, **processkwargs)
  File "/home/circleci/project/scripts/convert.py", line 238, in process_notebooks
    nbc.execute()
  File "/home/circleci/project/scripts/convert.py", line 97, in execute
    executor.preprocess(nb, {'metadata': {'path': self.path_only}})
  File "/home/circleci/project/venv/lib/python3.7/site-packages/nbconvert/preprocessors/execute.py", line 405, in preprocess
    nb, resources = super(ExecutePreprocessor, self).preprocess(nb, resources)
  File "/home/circleci/project/venv/lib/python3.7/site-packages/nbconvert/preprocessors/base.py", line 69, in preprocess
    nb.cells[index], resources = self.preprocess_cell(cell, resources, index)
  File "/home/circleci/project/venv/lib/python3.7/site-packages/nbconvert/preprocessors/execute.py", line 448, in preprocess_cell
    raise CellExecutionError.from_cell_and_msg(cell, out)
nbconvert.preprocessors.execute.CellExecutionError: An error occurred while executing the following cell:
------------------
print('Number of rows is', len(result))
------------------


TypeErrorTraceback (most recent call last)
<ipython-input-14-2eaef4d55702> in <module>
----> 1 print('Number of rows is', len(result))

TypeError: object of type 'NoneType' has no len()
TypeError: object of type 'NoneType' has no len()
```

**Comment by pllim** [comment]
For whatever reason, the query failed. I don't know why because I don't know where the full log is. You can see the logic in https://github.com/astropy/astroquery/blob/cfabf3a3d2de515b851730f26825f961d2bbc2d3/astroquery/vo_conesearch/conesearch.py#L82 

I cannot reproduce this failure locally.

**Comment by kakirastern** [comment]
Let me add a conditional later to cater for `NoneType` objects as well.

**Comment by eblur** [comment]
It looks like all the notebooks run now in [PR 449](https://github.com/astropy/astropy-tutorials/pull/449) but there is an error in the sphinx build:

```
Exception occurred:
  File "/home/circleci/project/venv/lib/python3.7/site-packages/matplotlib/sphinxext/plot_directive.py", line 262, in setup
    app.add_directive('plot', plot_directive, True, (0, 2, False), **options)
TypeError: add_directive() got an unexpected keyword argument 'alt'
The full traceback has been saved in /tmp/sphinx-err-t2luruki.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
make: *** [Makefile:43: html] Error 2

Exited with code exit status 2
```
